.github/CODEOWNERS

```diff
@@ -18,6 +18,7 @@
 /appengine/**/*                        @GoogleCloudPlatform/aap-dpes  @GoogleCloudPlatform/python-samples-reviewers
 /appengine/standard/django/**/*        @glasnt @GoogleCloudPlatform/aap-dpes  @GoogleCloudPlatform/python-samples-reviewers
 /appengine/flexible/django_cloudsql/**/*     @glasnt @GoogleCloudPlatform/aap-dpes  @GoogleCloudPlatform/python-samples-reviewers
+/appengine/standard_python3/bundled-services/**/*        @GoogleCloudPlatform/python-samples-reviewers
```

Comment:
Is there someone from your team or your GitHub team that we can add as the owner?

Reply:
@jinglundong LMK if you'd like help creating a GitHub team. 🙂

Reply:
**Update:** we can use the `@GoogleCloudPlatform/runtimes` team here.

@jinglundong cool if we add that team here?